tasks/gulp/compile-assets.js

```diff
@@ -28,6 +28,17 @@ const postcsspseudoclasses = require('postcss-pseudo-classes')({
 // check if destination flag is dist
 const isDist = taskArguments.destination === 'dist' || false
 
+// Set the destination
+const destinationPath = function () {
+  if (taskArguments.destination === 'dist') {
```

Comment:
This could use `isDist`

Reply:
Combined with another another condition. I think it makes it more obvious to explicitly show what is being tested

Reply:
I think using a variable to test this would be okay, especially as it's defined above.